### ISSUE TITLE
Add Documentation for Connecting to Chassis (Vagrant) Databases

### ISSUE THREAD

**Issue by kopepasah** [xwp/engineering-best-practices#21]
Connecting to an mysql database running in Chassis (or other Vagrant machine), requires some additional steps and connection process. This issue tracks that documentation.

**Comment by kopepasah** [comment]
@derekherman if this belongs in an alternate location, please let me know.

**Comment by derekherman** [comment]
@kopepasah Where are you wanting to add this information in the documentation itself?

**Comment by kopepasah** [comment]
Correct, as it is not documented anywhere. However, it may not be relevant if we are not using Chassis often.
